common.py

```diff
@@ -99,13 +106,20 @@ def LoadOrCreateConfig(config_file=None):
                os.path.expanduser('~/PCAutoBackup'))
   if not config.has_option('AUTOBACKUP', 'create_date_subdir'):
     config.set('AUTOBACKUP', 'create_date_subdir', '1')
+  if not config.has_option('AUTOBACKUP', 'default_interface'):
+    try:
+      config.set('AUTOBACKUP', 'default_interface',
+                 socket.gethostbyname(socket.gethostname()))
+    except socket.error:
+      logger.error('Unable to determine IP address. Please set manually!')
```

Comment:
Shouldn't this be logging.error(...)? Also, I think logging.warn(...) is more appropriate since you are setting the default value to localhost if you can't determine the IP automatically. Set the warning message to something like 'Unable to determine IP address. Using 127.0.0.1 instead.'